Restore the ability to send normal messages.
By Helmut Grohne <helmut@subdivi.de>, 2013-05-29

Normal messages are like headline messages except they are not discarded
when the contact is unavailable but queued instead.

See https://bugs.debian.org/710285

--- a/sendxmpp
+++ b/sendxmpp
@@ -49,7 +49,7 @@
 my $VERBOSE  = 0;
 my $DEBUG    = 0;
 # http://tools.ietf.org/html/rfc3921#section-2  section 2.1.1 - Types of Message
-my @suppported_message_types	= qw( chat error groupchat headline );
+my @suppported_message_types	= qw( chat error groupchat headline normal);
 my $message_type				= 'chat'; # default message type
 
 # start!
@@ -642,9 +642,9 @@
 
 Backward compatibility option. You should use B<--message-type=headline> instead. Send a headline type message (not stored in offline messages)
 
-=item B<--messages-type>
+=item B<--message-type>
 
-Set type of message. Supported types are: B<message chat headline>. Default message type is B<message>. Headline type message can be set also with B<--headline> option, see B<--headline>
+Set type of message. Supported types are: B<chat error groupchat headline normal>. Default message type is B<chat>. Headline type message can be set also with B<--headline> option, see B<--headline>
 
 =item B<-c>,B<--chatroom>
 
